x/did/types/diddoc_service.go

```diff
@@ -63,3 +65,62 @@ func IsUniqueServiceListByIdRule() *CustomErrorRule {
 		return nil
 	})
 }
+
+func IsSpecificDIDUrl(allowedNamespaces []string, pathRule, queryRule, fragmentRule ValidationType) *CustomErrorRule {
```

Comment:
Could you please back 3 new methods to validate.go?

Reply:
Done! ✅